sql/core/src/main/scala/org/apache/spark/sql/execution/WindowExec.scala

```diff
@@ -565,7 +566,7 @@ private[execution] abstract class WindowFunctionFrame {
 private[execution] final class OffsetWindowFunctionFrame(
     target: MutableRow,
     ordinal: Int,
```

Comment:
Since we are changing this file, can you add comment to explain what is ordinal (with an example)?